taxi/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.conf import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=63, unique=True)
+
+    class Meta:
+        verbose_name_plural = "drivers"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE, related_name="manufacturers")
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name="drivers")
```

Comment:
they are still incorrect

```suggestion
    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE, related_name="cars")
    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name="cars")
```

Reply:
Thanks for the point this :)